openpower/platform/Config.in

```diff
@@ -1,4 +1,4 @@
 config BR2_OPENPOWER_PLATFORM
     bool "Enable the OpenPower Platform"
     default n
-    depends on BR2_powerpc64
+    depends on BR2_powerpc64 || BR2_powerpc64le
```

Comment:
@jk-ozlabs - If we are going to deprecate BE support, should we remove the BR2_powerpc64 from this line?